sklearn/decomposition/truncated_svd.py

```diff
@@ -166,7 +172,15 @@ def fit_transform(self, X, y=None):
             # conventions, so reverse its outputs.
             Sigma = Sigma[::-1]
             U, VT = svd_flip(U[:, ::-1], VT[::-1])
-
+        elif self.algorithm == "lobpcg":
+            k = self.n_components
+            n_features = X.shape[1]
+            if k >= n_features:
+                raise ValueError("n_components must be < n_features;"
+                                 " got %d >= %d" % (k, n_features))
+            U, Sigma, VT = lobpcg_svd(X, self.n_components,
```

Comment:
This doesn't pass tol.

Reply:
Just as above. The new lobpcg code internally substitutes tol = .0 or None with a "reasonable" default, so there is no need to pass tol.